lib/sensors/stripe_sensor.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <cstdint>
+#include <optional>
+
+#include "io/gpio.hpp"
+#include <core/logger.hpp>
+#include <io/hardware_gpio.hpp>
+
+namespace hyped::sensors {
+class Keyence {
+ public:
+  Keyence(core::ILogger &log, const std::uint8_t newPin);
+  ~Keyence();
+
+  int getStripeCount();
+  void updateStripes();
+
+ private:
+  std::uint8_t pin;

```

Comment:
`pin_`. Look at the google C++ style guide if you're not sure but all class members like this should have should be snake case and have an underscore at the end (useful convention for identifying class members).